_gitbook/conventions/coding_style.md

```diff
@@ -0,0 +1,135 @@
+# Coding Style
+
+This style is used in the standard library. You can use it in your own project to make it familiar to other developers.
+
+## Naming
+
+__Type names__ are camelcased. For example:
+
+```crystal
+class ParseError < Exception; end
+
+module HTTP
+  class RequestHandler; end
+end
+
+alias NumericValue = Int32 | Int64 | Float32 | Float64
+
+lib LibYAML; end
+
+struct TagDirective; end
+
+enum Time::DayOfWeek; end
+```
```

Comment:
Thinking about it, whether is `;` is acceptable in real code is probably debatable, however the people who are okay with it are usually also okay with the long form of not using them. So avoiding them in the examples here might be less controversial for now.